Stripe/STPPaymentContext.m

```diff
@@ -568,7 +568,7 @@ - (void)requestSourceFlowRedirectPayment:(STPSource *)source {
     STPSourceCompletionBlock onRedirectCompletion = ^(STPSource *finishedSource, NSError *error) {
         stpDispatchToMainThreadIfNecessary(^{
             if (error) {
-                [self didFinishWithStatus:STPPaymentStatusError error:error];
+                [self didFinishWithStatus:STPPaymentStatusPending error:nil];
```

Comment:
ah, I see – I forgot that an SFVC error would end up here too. I think it's ok for now for an SFVC loading failure to result in a Pending status, but we should definitely try to fix that. Maybe we should have an STPRedirectContext error domain and check for that here – added to our todos.